### ISSUE TITLE
Multiplayer handicap

### ISSUE THREAD

**Issue by eai04191** [o5k/tetrio-issues#288]
### Is your feature request related to a problem? Please describe.
When I play TETR.IO with my friends, sometimes there is too much of a difference in ability. The same player always wins.

### Describe the solution you'd like
Allows to set a multiplier for the amount of garbage sent in an attack, **by player**.

### Describe alternatives you've considered
Allows the number of stocks to be set per player.

### Additional context
It's very boring when new players are always losing. This feature is desperately needed.

**Comment by csq-curiosity** [comment]
Some other possible solutions may be:
1. Adding room entry restriction settings, as described in #276.
2. Tuning override: __This is a setting currently available in tetrio__, where you can set ARE, line clear ARE, SDF and ARR on a room basis.
3. Adding "Piece for Piece" mode: Every player plays on the same pace, and the game puts more emphasis on efficacy. This may be a nightmare to implement on massive rooms, especially when there's someone with an unstable internet connection. (which is almost inevitable)
4. Adding teams mode: In case someone's skill is way above those of everyone else, majority vs (more skilled) minority, in a style of a boss raid, could be also done.

**Comment by eai04191** [comment]
Yes, I agree with the suggestion in #276, but that is different from the factors in this issue. I don't want to kick someone with a high skill set, I want to find a way to play with them.

> 4. Adding teams mode: In case someone's skill is way above those of everyone else, majority vs (more skilled) minority, in a style of a boss raid, could be also done.

Team mode is a good suggestion! But it doesn't work when it's 1v1. There will have to be other ways to do it.

**Comment by Volleo6144** [comment]
A few of my favorite video games have taken different approaches to handicaps, and all of them could reasonably translate to TETR.IO etc. These are ... essentially in order of what I think would make the best to worst implementations of handicaps:

- _Super Smash Bros. Melee_ and the Nintendo 64 original use knockback multipliers: a handicap of 1 means your character deals about three times less knockback to, and receives about three times as much knockback from, a character with a handicap of 9. This readily translates to adding individual garbage multipliers, and is what I would imagine is the best solution.
- _Tetris 99_'s local wireless mode allows players to start with KO badges, which increase your damage output (and also your countering ability) by up to 2x. Basically the same as above.
- _Super Smash Bros. Brawl_ and onwards allow you to set your character to start at something other than 0% (up to 300%). This ... wouldn't exactly make a good solution, especially for a game where damage can be removed such as TETR.IO; there's basically no middle ground between "pretty much impossible not to lose within 10s" and "essentially the same as zero".
- The 20XX mod of _Melee_, as well as the _Brawl_ mods _Project M_ and _Brawl−_, use varying stock counts. I imagine this would have a very similar issue to that of _Brawl_ and onwards if implemented in TETR.IO.
- _Puyopuyo Tetris_ just changes your gravity. (In Puyo, it instead changes the number of colors you get [Sweet/Mild = 3, Medium = 4, Hot/Spicy = 5] and in the case of Mild and Spicy, adds two garbage rows. Only on Spicy does the gravity change.) Wow, that does _such a good job_ at making it enjoyable to play with new players.

oh also team mode should exist anyway, even without the issue of handicaps

**Comment by eai04191** [comment]
This is a feature request and should not be closed, I think.

**Comment by eai04191** [comment]
no

**Comment by MNavoni** [comment]
Any updates on this? It's literally adding a garbage sent/recieved multiplier for every player in a custom lobby. and it would allow me to play with newbie friends.

**Comment by ZaptorZap** [comment]
**[Beta](https://tetr.io/about/patchnotes/#chlog_BETA_1_0_0)** introduced "mods" for Quick Play. They seem like a very natural and unique way for handicaps to exist in custom rooms, as opposed to just plainly making one player send less garbage.

**Comment by hedgeg0d** [comment]
sorry for duplicate. I tried to search in all issues with some keywords, but haven't found this one.  Good to see that i am not the only person who thought about this feature, anyway
